website/docs/docs/deploy/dbt-cloud-job.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Deploy with dbt Cloud"
+id: "dbt-cloud-job"
+description: "You can enable continuous integration (CI) to test every single change prior to deploying the code to production just like in a software development workflow."
+---
+
+Use dbt Cloud's job scheduler to streamline your data transformation workflows and empower data teams to run dbt in production with ease. 
+
+Running dbt in production using dbt Cloud's job scheduler eliminates the need to manually run dbt commands from the command line or set up an orchestration system. It's designed exclusively to streamline your dbt project deployments and runs, and keeps you data pipelines functioning seamlessly and efficiently.
+
+To experience the benefits of dbt Cloud, [sign up](https://cloud.getdbt.com/signup/) for a free account 
+today. 
+
+## Prerequisites
+
+- You must have a [dbt Cloud account](https://www.getdbt.com/signup/) and [Developer seat license](/docs/cloud/manage-access/seats-and-users)
+- You must have a dbt project connected to a [data platform](/docs/cloud/connect-data-platform/about-connections)
+- You must have [access permission](/docs/cloud/manage-access/about-access) to create, edit, and run jobs
+- You must set up a [deployment environment](/docs/collaborate/environments/dbt-cloud-environments) 
+- Your deployment environment must be on dbt version 1.0 or higher
+
+<Lightbox src ="/img/dbt-cloud-jobs.gif" width="85%" title="An overview of a dbt Cloud job run"/>
```

Comment:
This video feels like it should be rigth before "To experience the benefits of dbt Cloud..." above.

Reply:
ah! i originally had it after that sentence but then moved it down 🙈 - lesson learned: go w my gut!